scripts/nvbench_compare.py

```diff
@@ -226,16 +212,23 @@ def compare_benches(ref_benches, cmp_benches):
                     failure_count += 1
                     status = Fore.RED + "FAIL" + Fore.RESET
 
-                row.append(format_duration(ref_time))
-                row.append(format_percentage(ref_noise))
-                row.append(format_duration(cmp_time))
-                row.append(format_percentage(cmp_noise))
-                row.append(format_duration(diff))
-                row.append(format_percentage(frac_diff))
-                row.append(status)
+                if abs(frac_diff) >= threshold:
```

Comment:
This check could be done earlier by continuing after frac_diff is computed.

Reply:
I wanted to do it after all computation in case we want to expand to have different filter modes, but for KISS I can move it up